doc/Changelog.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
```

Comment:
If you do this you can give an arbitrary title to the page. Otherwise, doxygen names it after the filename. So for the change log is not needed.